packages/router/src/router.tsx

```diff
@@ -129,13 +129,17 @@ export function useHistory() {
 
 			await new Promise< void >( ( resolve ) => {
 				const classname = options.transition ?? '';
-				document.documentElement.classList.add( classname );
+				if ( classname ) {
```

Comment:
There's something I'm missing here. When can this happen?

There's a `! options.transition` check above that is meant to prevent using a transition entirely no?

Reply:
Agreed, nice find @jsnajdr. Updated in https://github.com/WordPress/gutenberg/pull/67378/commits/bb9b71fccd29daea4af9b4e39509bbab2322d1fb